docs/architecture/ingest-event-path.md

```diff
@@ -0,0 +1,164 @@
+# Path of an Event through Relay
+
+## Processing enabled vs not?
+
+Relay can run as part of a Sentry installation, such as within `sentry.io`'s
+infrastructure, or next to the application as a forwarding proxy. A lot of
+steps described here are skipped or run in a limited form when Relay is *not*
+running in processing mode:
+
+*  Event normalization does different (less) things.
+
+*  In `static` mode, project config is read from disk instead of fetched from
```

Comment:
Could we perhaps link to the modes description here?
Also, is "processing" also a mode, or more like, a feature? If the former, we should probably add it to the modes list.

Reply:
I want to keep this out of this document. I want to remove some of these modes once the original mode (fetching from sentry) is GA.